caribou/common/models/remote_client/aws_remote_client.py

```diff
@@ -646,7 +658,9 @@ def set_value_in_table_column(
             UpdateExpression=update_expression,
         )
 
-    def get_value_from_table(self, table_name: str, key: str, consistent_read: bool = True) -> tuple[str, float]:
+    def get_value_from_table(
+        self, table_name: str, key: str, consistent_read: bool = True, convert_from_bytes: bool = False
```

Comment:
While I understand why you did this like this, could we also cehck for the type and do this automatically? Basically if value type B -> decompress?

Reply:
That also sound like a good approach, since we are essentially only want the string representation of any table value anyways. I opened an issue for this approach: #341.